src/lib389/lib389/instance/setup.py

```diff
@@ -900,7 +900,12 @@ def _install_ds(self, general, slapd, backends):
         #  This is a little fragile, make it better.
         # It won't matter when we move schema to usr anyway ...
 
-        _ds_shutil_copytree(os.path.join(slapd['sysconf_dir'], 'dirsrv/schema'), slapd['schema_dir'])
+        shutil.copytree(
+            os.path.join(slapd["sysconf_dir"], "dirsrv/schema"),
+            slapd["schema_dir"],
+            copy_function=shutil.copy,
+            dirs_exist_ok=True,
```

Comment:
I think it's worth to add a small comment here that `dirs_exist_ok` is for bind-mounted volumes handling.

Reply:
Thanks, added.